spec/javascripts/components/cookie-banner-spec.js

```diff
@@ -63,8 +63,8 @@ describe('Cookie banner', function () {
     var cookieBannerMain = document.querySelector('.js-banner-wrapper')
     var cookieBannerConfirmation = document.querySelector('.gem-c-cookie-banner__confirmation')
 
-    expect(element).toBeVisible()
-    expect(cookieBannerMain).toBeVisible()
+    expect(element.offsetWidth > 0 || element.offsetHeight > 0).toBeTruthy()
```

Comment:
Jasmine jQuery uses jQuery's `visible` selector, which is basically this but has an [additional check](https://github.com/jquery/jquery/blob/a684e6ba836f7c553968d7d026ed7941e1a612d8/src/css/hiddenVisibleSelectors.js#L8) using `getClientRects`, is it worth adding this here?

Also since this repeats in a few places would it be a good idea/possible to create a function for this and call it?

Reply:
Good spot, I'll make those changes. Thanks @andysellick